### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by libklein** [keepassxreboot/keepassxc#7003] [Q&A]
Hi,

i was wondering whether keepassxc will "participate" in this year's hacktoberfest (https://hacktoberfest.digitalocean.com/). Essentially, this simply entails assigning the "hacktoberfest-accepted" label to eligible (i.e. "high quality") PRs, see https://hacktoberfest.digitalocean.com/faq.

**Comment by phoerious** [comment]
Hmm. Not against it. @droidmonkey ?

**Reply by droidmonkey** [reply]
Yup let's get a tag going and start tagging 2.7.0 issues we want worked. Could be a great push to releasing 2.7.0 in November. 

I'm updating our topics to include 'Hacktoberfest'

**Reply by droidmonkey** [reply]
I just tagged a bunch of issues with Hacktoberfest. Now to socialize this with the world...

**Reply by phoerious** [reply]
I think we have to register as maintainers, though: https://hacktoberfest.digitalocean.com/resources/maintainers

**Reply by droidmonkey** [reply]
Yup did that
